system/lib/symbols/wasm/libcompiler_rt.symbols

```diff
@@ -1,11 +1,125 @@
+-------- T __atomic_compare_exchange_2
+-------- T __atomic_compare_exchange_4
+-------- T __atomic_compare_exchange_8
+-------- T __atomic_exchange
+-------- T __atomic_exchange_1
+-------- T __atomic_exchange_16
+-------- T __atomic_exchange_2
+-------- T __atomic_exchange_4
+-------- T __atomic_exchange_8
+-------- T __atomic_fetch_add_1
+-------- T __atomic_fetch_add_16
+-------- T __atomic_fetch_add_2
+-------- T __atomic_fetch_add_4
+-------- T __atomic_fetch_add_8
+-------- T __atomic_fetch_and_1
+-------- T __atomic_fetch_and_16
+-------- T __atomic_fetch_and_2
+-------- T __atomic_fetch_and_4
+-------- T __atomic_fetch_and_8
+-------- T __atomic_fetch_or_1
+-------- T __atomic_fetch_or_16
+-------- T __atomic_fetch_or_2
+-------- T __atomic_fetch_or_4
+-------- T __atomic_fetch_or_8
+-------- T __atomic_fetch_sub_1
+-------- T __atomic_fetch_sub_16
+-------- T __atomic_fetch_sub_2
+-------- T __atomic_fetch_sub_4
+-------- T __atomic_fetch_sub_8
+-------- T __atomic_fetch_xor_1
+-------- T __atomic_fetch_xor_16
+-------- T __atomic_fetch_xor_2
+-------- T __atomic_fetch_xor_4
+-------- T __atomic_fetch_xor_8
+-------- T __atomic_load
+-------- T __atomic_load_1
+-------- T __atomic_load_16
+-------- T __atomic_load_2
+-------- T __atomic_load_4
+-------- T __atomic_load_8
+-------- T __atomic_store
+-------- T __atomic_store_1
+-------- T __atomic_store_16
+-------- T __atomic_store_2
+-------- T __atomic_store_4
+-------- T __atomic_store_8
+-------- T __clzti2
 -------- T __divdc3
+-------- T __divdi3
+-------- T __divmoddi4
 -------- T __divsc3
+-------- T __divtf3
+-------- T __divti3
+-------- T __eqtf2
+-------- T __extenddftf2
+-------- T __extendsftf2
+-------- T __fixdfdi
+-------- T __fixdfti
+-------- T __fixsfti
+-------- T __fixtfdi
+-------- T __fixtfsi
+-------- T __fixtfti
+-------- T __fixunsdfdi
+-------- T __fixunsdfti
+-------- T __fixunssfti
+-------- T __fixunstfdi
+-------- T __fixunstfsi
+-------- T __fixunstfti
+-------- T __floatdidf
+-------- T __floatditf
+-------- T __floatsitf
+-------- T __floattidf
+-------- T __floattisf
+-------- T __floattitf
+-------- T __floatunditf
+-------- T __floatunsitf
+-------- T __floatuntidf
+-------- T __floatuntisf
+-------- T __getf2
+-------- T __gttf2
+-------- T __letf2
+-------- T __lshrdi3
+-------- T __lshrti3
+-------- T __lttf2
+-------- T __moddi3
+-------- T __modti3
 -------- T __muldc3
+-------- T __muldi3
 -------- T __mulsc3
+-------- T __multc3
+-------- T __multf3
+-------- T __multi3
+-------- T __netf2
+         U __stack_pointer
+-------- T __subtf3
+-------- D __threwValue
+-------- T __trunctfdf2
+-------- T __trunctfsf2
+-------- T __udivdi3
+-------- T __udivmoddi4
+-------- T __udivmodti4
+-------- T __udivti3
+-------- T __umoddi3
+-------- T __umodti3
+-------- T __unordtf2
+-------- T __uremdi3
          U fmax
          U fmaxf
          U logb
          U logbf
+         U memcmp
+         U memcpy
          U scalbn
          U scalbnf
```

Comment:
I think this large change is actually expected since I forgot to do this in #9482

Reply:
Oh, ok, sounds good then.